src/gt4py/cartesian/config.py

```diff
@@ -83,3 +84,5 @@
 code_settings: Dict[str, Any] = {"root_package_name": "_GT_"}
 
 os.environ.setdefault("DACE_CONFIG", os.path.join(os.path.abspath("."), ".dace.conf"))
+
+DACE_DEFAULT_BLOCK_SIZE: str = os.environ.get("DACE_DEFAULT_BLOCK_SIZE", "32,1,1")
```

Comment:
That sounds a bit low. Unless we care about old hardware we can easily bump it to `64,8,1`

Reply:
Indeed it is, but since we have the liberty to reset a default, we can update it to something more likely. A part of me would like the system to crash if you are using an antic GPU ;)

Reply:
Both parts make sense ;)